File: ruby/delivery-api/GetTypes.rb
# DocSection: delivery_api_get_types
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kontent::Ai::Delivery::DeliveryClient.new project_id: '<YOUR_PROJECT_ID>'
delivery_client.types
               .limit(3)
               .execute do |response|
                 types = response.types
               end
# EndDocSection

File: ts/delivery-api/GetTypes.ts
// DocSection: delivery_api_get_types
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
import { createDeliveryClient } from '@kontent-ai/delivery-sdk';

const deliveryClient = createDeliveryClient({
  environmentId: '<YOUR_ENVIRONMENT_ID>
});

const response = await deliveryClient.types()
  .limitParameter(3)
  .toPromise();
// EndDocSection